agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/GeneToGeneOrthologyIndexer.java

```diff
@@ -22,12 +27,18 @@ public class GeneToGeneOrthologyIndexer extends Indexer {
 
 	private final GeneToGeneOrthologyDocumentInterface orthologyApi = RestProxyFactory.createProxy(GeneToGeneOrthologyDocumentInterface.class, ConfigHelper.getCurationApiUrl(), RestConfig.config);
 
+	private Set<String> allNeoGeneIDs;
+
 	public GeneToGeneOrthologyIndexer(IndexerConfig config) {
 		super(config);
 	}
 	
 	@Override
 	public void index() {
+		GeneRepository geneRepository = new GeneRepository();
+		allNeoGeneIDs = new HashSet<>(geneRepository.getAllGeneKeys());
+		geneRepository.close();
```

Comment:
These are already being pulled in the BaseService class.

Reply:
We were using services to get the records. And they can be found in the BaseService so you could `BaseService base = new BaseService();` then you would need to change the three methods: getAllNeoAlleleIDs, getAllNeoGeneIDs, getAllNeoModelIDs to public.